test/e2e/sagemaker/tests/test_transformjob.py

```diff
@@ -0,0 +1,154 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+#	 http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+"""Integration tests for the SageMaker TransformJob API.
+"""
+
+import boto3
+import pytest
+import logging
+from typing import Dict
+import time
+
+from sagemaker import SERVICE_NAME, service_marker, CRD_GROUP, CRD_VERSION
+from sagemaker.replacement_values import REPLACEMENT_VALUES
+from common.resources import load_resource_file, random_suffix_name
+from common import k8s
+from common.aws import get_aws_region
+
+RESOURCE_PLURAL = 'transformjobs'
+
+
+@pytest.fixture(scope="module")
+def sagemaker_client():
+    return boto3.client('sagemaker')
+
+
+@pytest.fixture(scope="module")
+def xgboost_transformjob(sagemaker_client):
+    #Create model using boto3 for TransformJob
+    model_name = random_suffix_name("xgboost-model", 32)
+
+    create_response = sagemaker_client.create_model(ModelName=model_name,
+                        PrimaryContainer={
+                                        'Image': REPLACEMENT_VALUES["XGBOOST_IMAGE_URI"],
+                                        'ModelDataUrl': REPLACEMENT_VALUES["TRANSFORM_MODEL"],
+                                        'Environment': {}
+                                    },
+                        ExecutionRoleArn=REPLACEMENT_VALUES["SAGEMAKER_EXECUTION_ROLE_ARN"]
+                       )
+    #logging.debug(create_response)
+
+    #Check if the model is created successfully
+    describe_model_response = sagemaker_client.describe_model(ModelName=model_name)
+    assert describe_model_response["ModelName"] is not None
+    
+    resource_name = random_suffix_name("xgboost-transformjob", 32)
+
+    #Use the model created above
+    replacements = REPLACEMENT_VALUES.copy()
+    replacements["MODEL_NAME"] = model_name
+    replacements["TRANSFORM_JOB_NAME"] = resource_name
+
+    transformjob = load_resource_file(
+        SERVICE_NAME, "xgboost_transformjob", additional_replacements=replacements)
+    #logging.debug(transformjob)
+
+    # Create the k8s resource
+    reference = k8s.CustomResourceReference(
+        CRD_GROUP, CRD_VERSION, RESOURCE_PLURAL, resource_name, namespace="default")
+    resource = k8s.create_custom_resource(reference, transformjob)
+    resource = k8s.wait_resource_consumed_by_controller(reference)
+
+    assert resource is not None
+
+    yield (reference, resource)    
+
+
+@service_marker
+@pytest.mark.canary
+class TestTransformJob:
+    def _get_created_transformjob_status_list(self):
+        return ["InProgress", "Completed"]
```

Comment:
`Completed` should be in _get_stopped_transformjob_status_list rather than this one

Reply:
Fixed